test/src/test/java/org/corfudb/infrastructure/log/StreamLogFilesTest.java

```diff
@@ -40,8 +40,12 @@ public void testWriteReadWithChecksum() {
         assertThat(log.read(address0).getPayload(null)).isEqualTo(streamEntry);
 
         // Disable checksum, then write and read then same entry
-        log = new StreamLogFiles(getDirPath(), true);
-        log.append(address0, new LogData(DataType.DATA, b));
+        // An overwrite exception should occur, since we are writing the
+        // same entry.
```

Comment:
1. the test is named "WithChecksum", so why are we writing/reading here with checksum disabled?

2. Indeed, as @no2chem indicated, why are we trying to overwrite the same entry? is this intentional?